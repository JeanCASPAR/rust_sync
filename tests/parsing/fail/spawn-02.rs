use rust_sync::sync;

sync! {
    #![pass(0)]

    node oui(c : bool) = ()
    where
        a : int = spawn c;
}

fn main() {}
