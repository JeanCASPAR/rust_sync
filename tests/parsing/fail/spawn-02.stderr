error: expected parentheses
 --> tests/parsing/fail/spawn-02.rs:8:26
  |
8 |         a : int = spawn c;
  |                          ^
